editbl/R/eDT.R

```diff
@@ -241,9 +245,11 @@ eDTServer <- function(
     statusColor = c("insert"="#e6e6e6", "update"="#32a6d3", "delete"="#e52323"),
     inputUI = editbl::inputUI,
     defaults = tibble(),
+    cloneDefaults = c(),
```

Comment:
```suggestion
    cloneDefaults = tibble(),
```

So, as I understand it now, you are selecting a subset of the columns of `defaults` argument to be used when cloning.

I had in mind the more generic implementation of having the `cloneDefaults` being a tibble  on it's own.
Where in 99% of the cases it will just equal `defaults` or be a subset thereof though. However, this way you can also support:

1. Additional columns with defaults when cloning vs inserting
2. Different values for cloned columns vs inserting

A potential use case could be that you want to keep track on whether or not  a row was cloned with a column `is_cloned`. In this case the default for adding rows should be `FALSE` whereas on cloning it is `TRUE`. And maybe you disable it from being modified by the user.

Technically we can also do an implementation where both a `tibble()` and a `vector` are supported as input. But since the former should support all use cases of the latter, I would not struggle with also supporting the second one.

But I'm open for discussion on this if you see advantages with the current approach!